lib/runcible/instance.rb

```diff
@@ -72,14 +84,6 @@ def accessible_class(class_object)
       #  (e.g. consumer_group)
       class_object.name.split('::').last.underscore
     end
-
-    def self.gather_classes(type)
```

Comment:
No, this is [UselessAccessModifier](http://www.rubydoc.info/github/bbatsov/rubocop/Rubocop/Cop/Lint/UselessAccessModifier). Another option would be to move the method out of the private area and use `private_class_method :gather_classes `. I lean slightly more to this solution in case we add more private class methods.